ec2api/api/route_table.py

```diff
@@ -480,18 +480,26 @@ def _update_routes_in_associated_subnets(context, route_table, cleaner,
 
 def _update_subnet_host_routes(context, subnet, route_table, cleaner=None,
                                rollback_route_table_object=None,
-                               router_objects=None, neutron=None):
+                               router_objects=None, neutron=None,
+                               vpc_route_revert=False, add_vpc_route=False):
```

Comment:
I was referring to call to this function from route_table API like associate_route_table where they are not passing this argument so there will be changed behavior.